tidb-cloud/tidb-cloud-release-notes.md

```diff
@@ -8,6 +8,29 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## July 27, 2023
+
+**General changes**
+
+- Introduce a powerful feature in TiDB Cloud [Data Service](https://tidbcloud.com/console/data-service) - Automatic Endpoint Generation.
+
+    Developers can now effortlessly create HTTP endpoints with minimal clicks and configurations. Eliminate repetitive boilerplate code, simplify and accelerate endpoint creation, and reduce potential errors.
+
+    For more information on how to use this feature, see [Generate an endpoint automatically](/tidb-cloud/data-service-manage-endpoint.md#generate-an-endpoint-automatically).
+
+- Support `PUT` and `DELETE` request methods for endpoints in TiDB Cloud [Data Service](https://tidbcloud.com/console/data-service).
+
+    - You can use the `PUT` method to update or modify data, such as an `UPDATE` statement.
```

Comment:
Use the PUT method to update or modify data, similar to an UPDATE statement.
Use the DELETE method to delete data, similar to a DELETE statement.